### PR TITLE
Add train_amp_python

### PR DESCRIPTION
**Pull request by zhangbo9674**
训推一体：添加混合精度训练文档。

- tipc目录预览：https://github.com/zhangbo9674/models/blob/dev/amp_train_doc_new/tutorials/tipc/train_amp_infer_python/train_amp_infer_python.md

- mv3 doc目录预览：https://github.com/zhangbo9674/models/blob/dev/amp_train_doc_new/tutorials/mobilenetv3_prod/Step6/docs/train_amp_infer_python.md

### CODE REVIEW

File: tutorials/tipc/train_amp_infer_python/train_amp_infer_python.md [hunk 1]
```diff
@@ -2,6 +2,286 @@
...
+**【实战】**
+
+MobileNetv3中添加O2混合精度训练如下所示，参考链接：[train.py](https://github.com/PaddlePaddle/models/blob/release/2.2/tutorials/mobilenetv3_prod/Step6/train.py)。
+
+1）AMP-O1：
+
+训练代码执行脚本如下：
+
+```
+GLOG_v=5 python3.8 train.py --data-path=./ILSVRC2012 --lr=0.1 --batch-size=256 --amp_level=O1
```
Comment (littletomatodonkey) [inline tutorials/tipc/train_amp_infer_python/train_amp_infer_python.md]: python3.8改成python3吧

File: tutorials/tipc/train_amp_infer_python/train_amp_infer_python.md [hunk 2]
```diff
@@ -2,6 +2,286 @@
...
+
+- 第18行：表明`conv2d`实际执行的kernel类型为FP16
+
+
+2）AMP-O2：
+
+训练代码执行脚本如下：
+
+```
+GLOG_v=5 python3.8 train.py --data-path=./ILSVRC2012 --lr=0.1 --batch-size=256 --amp_level=O2
```
Comment (littletomatodonkey) [inline tutorials/tipc/train_amp_infer_python/train_amp_infer_python.md]: 同python3

File: tutorials/tipc/train_amp_infer_python/train_amp_infer_python.md (hunk 1 above)
Comment (zhangbo9674) [reply tutorials/tipc/train_amp_infer_python/train_amp_infer_python.md]: done，tks.

File: tutorials/tipc/train_amp_infer_python/train_amp_infer_python.md (hunk 2 above)
Comment (zhangbo9674) [reply tutorials/tipc/train_amp_infer_python/train_amp_infer_python.md]: done, tks.

**Review by littletomatodonkey — APPROVED** [review]

**Pull request merged**